### PR TITLE
sphinx conversion -- sdk module

### PR DESCRIPTION
**Pull request by tconsigny**

### CODE REVIEW

File: verticapy/sdk/vertica/udf/gen.py [hunk 1]
```diff
@@ -74,6 +88,23 @@ def generate_lib_udf(
     -------
     udx_str, sql
         UDF py file, str needed to install the library.
+
+    Example
+    -------
+    .. ipython:: python
+
+        from verticapy.udf import generate_lib_udf
```
Comment (oualib) [inline verticapy/sdk/vertica/udf/gen.py:96]: from verticapy.sdk.vertica.udf import generate_lib_udf

File: verticapy/sdk/vertica/udf/gen.py [hunk 2]
```diff
@@ -74,6 +88,23 @@ def generate_lib_udf(
...
+        @suppress
+        import math
+
+        udx_str, udx_sql = generate_lib_udf([(math.exp, [float], float, {}, "python_exp"),
```
Comment (oualib) [inline verticapy/sdk/vertica/udf/gen.py:100]: please use PEP8 formatting:
```
udx_str, udx_sql = generate_lib_udf(
    [(math.exp, [float], float, {}, "python_exp"), (math.isclose, [float, float], bool, {"abs_tol": float}, "python_isclose"),],
    library_name = "python_math",
    file_path = "",
    create_file = False
)
```
@mail4umar or @afard is it possible to use black on RSTs? Specifically for python cells.

File: verticapy/sdk/vertica/udf/gen.py [hunk 3]
```diff
@@ -74,6 +88,23 @@ def generate_lib_udf(
...
+                                             (math.isclose, [float, float], bool, {"abs_tol": float}, "python_isclose"),],
+                                              library_name = "python_math",
+                                              file_path = "",
+                                              create_file = False)
+        print(udx_str)
```
Comment (oualib) [inline verticapy/sdk/vertica/udf/gen.py:105]: is it possible to separate it into two blocks so the user can clearly see the result?

Ex:
VerticaPy will generate the UDX Python code:
```
 print(udx_str)
```
And the SQL needed to install the function:
```
print("\\n".join(udx_sql))
```

File: verticapy/sdk/vertica/udf/load.py [hunk 4]
```diff
@@ -40,24 +40,38 @@ def import_lib_udf(
...
+
+                            Example: {"param1": int,
+                            "param2": str}
+
+        **new_name**     :
+                            [str] New   function   name  when
+                            installed in Vertica.
+
     library_name: str
         Library Name.
```
Comment (oualib) [inline verticapy/sdk/vertica/udf/load.py:76]: is there any example for this function?

**Review by oualib — CHANGES_REQUESTED** [review]
good work. We need to change few things before merging this PR.

**Comment by oualib** [comment]
@mail4umar can you please complete my requests?

File: verticapy/sdk/vertica/udf/load.py (hunk 4 above)
Comment (tconsigny) [reply verticapy/sdk/vertica/udf/load.py:76]: I didn't see any on the current website (https://www.vertica.com/python/documentation_last/udf/)

File: verticapy/sdk/vertica/udf/load.py [hunk 5]
```diff
@@ -40,24 +40,38 @@ def import_lib_udf(
...
     include_dependencies: str / list, optional
```
Comment (oualib) [inline verticapy/sdk/vertica/udf/load.py:77]: Example:

```
Example
--------
We import the math module:

.. code-block:: python

   import math

We will import two functions from this module: `exp` and `is_close`.

.. important:: Python is type-agnostic, while Vertica requires specific data types. It's crucial to specify input, output, and parameter types when generating User-Defined Extensions (UDx). These functions will be automatically installed, allowing you to call them directly using SQL.

.. code-block:: python

        from verticapy.sdk.vertica.udf import import_lib_udf

        import_lib_udf(
            [
                (math.exp, [float], float, {}, "python_exp"), 
                (math.isclose, [float, float], bool, {"abs_tol": float}, "python_isclose"),
            ],
            library_name = "python_math"
        )

.. important::  In this example, we're utilizing a standard Python function. If you wish to use a non-standard one, you'll need to install it on each node individually.
```

**Review by oualib — CHANGES_REQUESTED** [review]
some changes to make.

**Review by oualib — APPROVED** [review]

**Pull request merged**